package/yast2-network.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu May  4 11:21:13 UTC 2023 - Knut Anderssen <kanderssen@suse.com>
+
+- Display information about firmware configured interfaces and
+  prevent the forbid to edit them (bsc#1209589)
```

Comment:
The message looks confusing to me (the second part). What do you mean?

Reply:
even to me xxDDD

Reply:
```suggestion
  forbid to edit them (bsc#1209589)
```